code/_core/obj/item/fishing/rod.dm

```diff
@@ -26,6 +26,7 @@
 	var/catch_time
 
 	var/next_tick = 0
+	var/nospam
```

Comment:
Instead of no_spam, you instead should use
```
INTERACT_CHECK
INTERACT_DELAY
```
before the code executes.

Reply:
reasons for that are in the #contributing channel of the discord, do tell if more changes are required tho